lib/shopify_api/message_enricher.rb

```diff
@@ -14,7 +14,7 @@ def message
             elsif parsed_body['errors']
               Array(parsed_body['errors']).join('; ')
             end
-          rescue JSON::ParserError
+          rescue JSON::ParserError, TypeError
```

Comment:
I think catching `TypeError` is a bit dangerous because we may end up masking other errors. When the errors are an array, can we just loop over them and join them as we do with the elements of `errors`?